application/app/src/main/java/uk/nhs/digital/uec/api/service/impl/FuzzyServiceSearchService.java

```diff
@@ -6,6 +6,7 @@
 import org.springframework.stereotype.Service;
 import uk.nhs.digital.uec.api.model.DosService;
 import uk.nhs.digital.uec.api.service.FuzzyServiceSearchServiceInterface;
+import uk.nhs.digital.uec.api.utils.MockDosServicesUtil;
 
 @Profile("prod")
```

Comment:
What is the meaning of the `prod` profile in this context?

Reply:
It's badly named. I will resolve.